scripts/sipify.pl

```diff
@@ -133,8 +133,17 @@ sub processDoxygenLine {
     if ( $line =~ m/\\since .*?([\d\.]+)/i ) {
         return ".. versionadded:: $1\n";
     }
-    if ( $line =~ m/\\see (.*)/ ) {
-        return ".. seealso:: $1\n";
+    if ( $line =~ m/\\see *(.*)/ ) {
+        my $seealso = $1;
+        if (  $1 =~ m/^Qgs[A-Z]/ ) {
+            return ".. seealso:: :py:class:`$seealso`\n";
+        }
+        elsif (  $1 =~ m/^[a-z]/ ) {
```

Comment:
I suspect you should need to detect that there's no '::' here

Reply:
good catch